sshkey.c

```diff
@@ -723,6 +725,7 @@ sshkey_sk_cleanup(struct sshkey *k)
 static int
 sshkey_prekey_alloc(u_char **prekeyp, size_t len)
 {
+#ifdef HAVE_MMAP
```

Comment:
you can have mmap() but not MAP_ANON or MAP_PRIVATE, so it needs to check those too (see openbsd-compat/arc4random.h) for an example, (although it doesn't check HAVE_MMAP and probably should).